testing/scenario_app/run_ios_tests.sh

```diff
@@ -33,9 +33,15 @@ if [[ $# -eq 1 ]]; then
   FLUTTER_ENGINE="$1"
 fi
 
-cd ios/Scenarios
-set -o pipefail && xcodebuild -sdk iphonesimulator \
-  -scheme Scenarios \
-  -destination 'platform=iOS Simulator,name=iPhone 8' \
-  test \
-  FLUTTER_ENGINE="$FLUTTER_ENGINE"
+echo "Scenarios tests are currently disabled due to flakiness"
+echo "See: https://github.com/flutter/flutter/issues/61620"
+
+# TODO(cbracken): re-enable when
+# https://github.com/flutter/flutter/issues/61620 is fixed.
```

Comment:
I don't think you need to do this. This disables all our integration tests on iOS whereas there are still XCUITests in there (not the same target as the one above).

Reply:
Oh interesting -- alright taking a look and will patch those.

Reply:
Hrm -- looking at this, it's unclear to me which subset of the tests you mean. The new one isn't a lifecycle test but an initial route test. Are you suggesting disabling them one by one until we get it consistently passing?

Reply:
Updated to just disable the newly failing one.